repos/system_upgrade/common/actors/livemode/modify_userspace_for_livemode/libraries/prepareliveimage.py

```diff
@@ -0,0 +1,514 @@
+import grp
+import os
+import os.path
+from distutils.version import LooseVersion
```

Comment:
@MichalHe  this can be resolved in the same way as 236483d5478ce845726ee05b7ca3e080105cdb80

Reply:
Done.